app/services/statements/query.rb

```diff
@@ -1,32 +1,66 @@
 module Statements
   class Query < Query
-    attr_reader :lead_provider, :cohort_start_years, :updated_since, :state, :output_fee
+    attr_reader :scope
 
-    def initialize(lead_provider: nil, cohort_start_years: nil, updated_since: nil, state: nil, output_fee: true)
```

Comment:
maybe we can have a test to show that with this implementation we could pass in `nil` values for `null` queries?

Reply:
Yeah I'm rewriting the tests to take account of the new `where_` methods. Then we just need to ensure `initialize` calls them, which should reduce doubling up.

They're kind of also testing Postgres at the moment which seems a bit odd.

Reply:
I just made them private until we might want to use them individually.